api/src/team/infrastructure/repositories/certification-center-membership.repository.js

```diff
@@ -63,6 +63,25 @@ const countActiveMembersForCertificationCenter = async function (certificationCe
   return count;
 };
 
+/**
+ * @param {Object} params
+ * @param {string} params.certificationCenterId
+ * @param {date} params.disabledAt
+ * @param {string} params.updatedByUserId
+ */
+
+const disableMembershipsByCertificationCenterId = async function ({
```

Comment:
Supprimer cette ligne vide : 
```suggestion
/**
 * @param {Object} params
 * @param {string} params.certificationCenterId
 * @param {date} params.disabledAt
 * @param {string} params.updatedByUserId
 */
const disableMembershipsByCertificationCenterId = async function ({
```